runtime/helpers/axiosHelper.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+const { SERVER } = require("../../shared-objects/servers");
+const axios = require("axios");
+
+const errHandler = err => {
+    // console.error(err)
+    if (err.response && err.response.data) {
+        // console.error('API response', err.response.data)
+        throw err.response.data
+    }
+    throw err
+}
+
+
+class Api {
+    service(jwt) {
+        return axios.create({
+            baseURL: SERVER.URL,
+            timeout: 15000,
+            headers: { Authorization: jwt },
+            withCredentials: true,
+        })
+    }
+    getStudentsAsAdmin(jwt) {
```

Comment:
Please move all admin-student related methods to its own adminStudentApi.js class/module.